src/components/MemberCard.tsx

```diff
@@ -0,0 +1,95 @@
+'use client'
+
+import React from 'react'
+import styled from '@emotion/styled'
+import { FontVariant, Color } from '@/app/theme'
+import { ImageWithFallback } from '@/components/ImageWithFallback'
+import type { IMember } from '@/data/members'
+
+const Card = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  align-items: center;
+  width: 200px;
+`
+
+const Info = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 6px;
+  align-items: center;
+  ${FontVariant.body_md}
+  width: fit-content;
+`
+
+const Email = styled.span`
+  color: ${Color.gray700};
+  ${FontVariant.body_md}
+`
+
+const NameWithWebsite = styled.a`
+  ${FontVariant.body_lg}
+  color: ${Color.gray900};
+`
+
+const ImageContainer = styled.div`
+  position: relative;
+  width: 100%;
+  padding-top: 100%;
+  border-radius: 50%;
+  overflow: hidden;
+`
+
+const MemberImage = styled(ImageWithFallback)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`
+
+const Name = styled.span`
+  ${FontVariant.body_lg}
+  color: ${Color.gray900};
+`
+
+const Affiliation = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 6px;
+  color: ${Color.gray700};
+  }
+`
+
+export default function MemberCard({ mem }: { mem: IMember }) {
```

Comment:
Let's be consistent with the exporting style (export default vs export const).
I prefer the latter.